EngineMath/Vector2D.h

```diff
@@ -0,0 +1,22 @@
+#pragma once
```

Comment:
The Vector2D and 3D classes are still present in source, but removed from the solution. Why?